cmd/sqlflowserver/main_test.go

```diff
@@ -57,6 +57,15 @@ func AssertEqualAny(a *assert.Assertions, expected interface{}, actual *any.Any)
 	}
 }
 
+func AssertGreaterEqualAny(a *assert.Assertions, actual *any.Any, expected interface{}) {
```

Comment:
I don't think the following changes are relevant to this PR.

Reply:
I updated the description of this PR, this change try to fix randomly fails in PR CI check, or pull requests would take several re-runs to pass the checks.